pkgs/by-name/go/gosmee/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGoModule rec {
+  name = "gosmee-${version}";
+  version = "0.21.0";
+  rev = "v${version}";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "chmouel";
+    repo = "gosmee";
+    sha256 = "sha256-VNBz6roula/YlsYMM1kjQT3TLuQ86/MESzNNkEf/92M=";
+  };
+  vendorHash = null;
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postUnpack = ''
+    printf ${version} > $sourceRoot/gosmee/templates/version
+  '';
```

```suggestion
  postPatch = ''
    echo ${version} > gosmee/templates/version
  '';
```
changing files should be done in patchPhase